src/sidebar/components/test/tag-editor-test.js

```diff
@@ -66,118 +95,166 @@ describe('TagEditor', function() {
     });
   });
 
-  it("creates a `list` prop on the input that matches the datalist's `id`", () => {
+  it('generates an ordered datalist containing the array values returned from filter()', () => {
```

Comment:
It wasn't immediately obvious to me what `filter()` refers to here. I presume you mean the tags service?